### PR TITLE
docs: update release docs.

### PR DESCRIPTION
**Pull request by jcvenegas**
- Add release bump pipeline
- Add command to tag this repo.

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: release/release.md [hunk 1]
```diff
@@ -122,8 +129,14 @@ make sure the packages install and work. To help with this you can use the [pack
 If you found any issue during the release process and you fix it, please send it back.
 After your changes are merged, tag Kata packaging with `${NEW_VERSION}` to identify the code used for the release.
 
+   ```bash
+   $ git tag -a "${NEW_VERSION}" -s -m "Kata Containers release ${NEW_VERSION}"
+   $ git push git@github.com:kata-containers/packaging.git 1.8.0-rc0
```
Comment (jodh-intel) [inline release/release.md]: Shouldn't this line be:

```sh
$ git push git@github.com:kata-containers/packaging.git "${NEW_VERSION}"
```

File: release/release.md [hunk 2]
```diff
@@ -58,6 +58,13 @@ $ cd ${GOPATH}/src/github.com/kata-containers/packaging
 $ git pull
 ```
 1. Bump repositories
+
+   Trigger a [CI job][bump-job] to bump Kata
+
+   http://jenkins.katacontainers.io/blue/organizations/jenkins/release
```
Comment (jodh-intel) [inline release/release.md]: This link is the same as the `bump-job` one, so I think you can remove this line?

File: release/release.md [hunk 3]
```diff
@@ -58,6 +58,13 @@ $ cd ${GOPATH}/src/github.com/kata-containers/packaging
...
+
+   or
```
Comment (jodh-intel) [inline release/release.md]: I know it's obvious, but for absolute clarity I'd still change this to be:

> or run the following commands:

**Comment by egernst** [comment]
@jcvenegas any updates here ?

**Comment by jodh-intel** [comment]
Ping @jcvenegas - could you tal at fixing this week?

**Comment by amshinde** [comment]
@jcvenegas Can you take a look at this?

**Comment by jodh-intel** [comment]
@jcvenegas - could you update? Would be great to get this landed (doc PRs are *more important* than codes ones imho :)

**Comment by amshinde** [comment]
ping @jcvenegas

**Comment by chavafg** [comment]
ping @jcvenegas

**Comment by chavafg** [comment]
@jcvenegas can you address comments above?

**Comment by jodh-intel** [comment]
Re-ping @jcvenegas :smile:

**Comment by jodh-intel** [comment]
@jcvenegas - just a few updates based on review feedback and we should be able to land this...

**Comment by jodh-intel** [comment]
@jcvenegas - your 6 month reminder... :smile:

**Pull request closed without merging**